setup.cfg

```diff
@@ -1,3 +1,45 @@
+[metadata]
+name = your_package_name
+author = PDS
```

Comment:
It wasn't possible to do `version = attr: versioneer.get_version()` or something like that here?

Reply:
To be honest, I only poked around attempting to make `get_cmdclass()` to work before stumbling onto a ticket indicating that it couldn't. Since it had to be in `setup.py` I just put both the versioneer calls there for consistency. I can try moving the version stuff back to `setup.cfg`.

Reply:
Sounds great to me @MJJoyce 🎉

Reply:
Looks like this chokes when we move it to `setup.cfg`.

Adding this to `setup.cfg`
```
version = attr: versioneer.get_version()
```

```
    File "/private/var/folders/fq/z23zw0pd18x29jndfz5tqwqntzw8gz/T/pip-build-env-dxb_kyxd/overlay/lib/python3.9/site-packages/setuptools/config.py", line 557, in _parse_version
      version = self._parse_attr(value, self.package_dir)
    File "/private/var/folders/fq/z23zw0pd18x29jndfz5tqwqntzw8gz/T/pip-build-env-dxb_kyxd/overlay/lib/python3.9/site-packages/setuptools/config.py", line 392, in _parse_attr
      return getattr(module, attr_name)
  AttributeError: module 'versioneer' has no attribute 'get_version()'
```